icu4c/packaging/distrelease.ps1

```diff
@@ -22,6 +22,9 @@
 #  see https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-5.1&viewFallbackFrom=powershell-Microsoft.PowerShell.Core 
 #    for more about execution policies.
 
+Param(
+  [string]$architecture = $( Read-Host "Enter architecture" )
```

Comment:
Can we maybe assume a default of x64?
Since I think that is what the script used to do if you didn't provide any arguments. (Before we added support for x86)

Also, more of a nit, but perhaps just `arch` to save typing.

Reply:
Sure!